MAUI/PDF-Viewer/Signature.md

```diff
@@ -115,6 +115,34 @@ PdfViewer.AddAnnotation(customStamp);
 
 N> To add a text signature, you can use an image containing the signature text . You can then add it in the same manner as an image signature.
 
+## SignatureCreated event
```

Comment:
This topic can be moved at last after the signature model view. So that user will have knowledge about the signature pad.

<img width="480" height="422" alt="image" src="https://github.com/user-attachments/assets/cfdcd6ac-2e1c-40d7-b440-40b8e940d445" />